e2e-tests/tfm/cypress/e2e/pages/amendments/amendmentsPage.js

```diff
@@ -49,6 +59,59 @@ const amendmentsPage = {
     amendmentsPage.continueAmendment().click();
   },
 
+  navigateToIsUsingFacilityEndDatePage: (startNewAmendment, changeFacilityValue, newCoverEndDate) => {
+    facilityPage.facilityTabAmendments().click();
+
+    if (startNewAmendment) {
+      amendmentsPage.addAmendmentButton().should('exist');
+      amendmentsPage.addAmendmentButton().contains('Add an amendment request');
+      amendmentsPage.addAmendmentButton().click();
+    } else {
+      amendmentsPage.continueAmendmentButton().click();
+    }
+
+    cy.url().should('contain', 'request-date');
+    amendmentsPage.amendmentRequestDayInput().clear().type(dateConstants.todayDay);
```

Comment:
can todayDay etc be dereferenced at the top since used many times

Reply:
sure have changed